shared/video-sdk/develop/stream-raw-audio-and-video/project-implementation/windows.mdx

```diff
@@ -20,303 +20,251 @@ To enable or disable processing of captured raw video data, add a button to the
 
 ### Handle the system logic
 
-This section describes the steps required to import libraries, declare variables, and set
-  up access to UI elements.
+This sections describes the steps required to use the relevant libraries and declare the necessary variables, and set up access to the UI elements.
 
 1.  **Define a variable to manage video processing**
 
-        In **Solution Explorer**, open `AgoraImplementationDlg.cpp` and define a global zoom variable:
+        From **Solution Explorer**, open `AgoraImplementationDlg.cpp` and add the following variable after the list of header includes:
         
         ```cpp
         bool isZoomed = false;
         ```
-2.    **Start and stop video processing**
+2.  **Start and stop video processing**
     
-        When a user presses `Zoom In`, your app captures the video frame and processes it for zooming. To implement
-  the UI for this workflow, in **Dialog Editor**, double-click **Zoom In**. **Dialog Editor** automatically creates
-  and opens an event listener for you. Add the following code to this event listener:
-   
-        ```cpp
-        // Toggle the zoom state
-        isZoomed = !isZoomed;
-
-        // Get a handle to the parent control of the button
-        CWnd* pParent = GetDlgItem(IDC_BUTTON4);
+    When the user presses the **Zoom In** button, the current state of the `isZoomed` variable is updated. This variable controls the behavior of the `onCaptureVideoFrame` callback, either enabling or disabling frame cropping. To implement this workflow, double-click on the **Zoom In** button in the **Dialog Editor**, which creates an event listener for you. Add the following code to the event listener:
+    ```cpp
+    // Toggle the zoom state and get a handle to the button control
+    isZoomed = !isZoomed;
+    // We implemented the Agora Windows Get Started guide, using the button with ID 'IDC_BUTTON3' for zoom in and zoom out functionality. However, please note that the button ID may differ in your specific scenario.
+    CButton* pButtonCtrl = static_cast<CButton*>(GetDlgItem(IDC_BUTTON3));
+
+    // Set the button text based on the zoom state
+	pButtonCtrl->SetWindowText(isZoomed ? L"Zoom Out" : L"Zoom In");
```

Comment:
Here are no extra benefit to use  ternary conditional operator (?:) . Even it is difficult to debug at some point. As I have needed to debug while testing the app. With the if-else statement, it's easier to set breakpoints and examine the state of the program at each branch. With the ternary operator, it can be more difficult to isolate specific conditions while debugging. No point to make these changes.